go.mod

```diff
@@ -20,22 +19,23 @@ require (
 	github.com/spf13/cobra v1.8.0
 	github.com/stretchr/testify v1.8.4
 	github.com/vektra/errors v0.0.0-20140903201135-c64d83aba85a
-	golang.org/x/exp v0.0.0-20240112132812-db7319d0e0e3
-	golang.org/x/net v0.20.0
+	golang.org/x/exp v0.0.0-20240222234643-814bf88cf225
```

Comment:
If/when doing any updates on dependencies, please submit a separate PR.

Reply:
was not intentional, result of `go mod tidy`